backend/app/routes/index.js

```diff
@@ -7,7 +7,7 @@ const broadcast = require('./broadcast');
 const faq = require('./faq/post');
 const getFaq = require('./faq/getFaq');
 const deleteFaq = require('./faq/deleteFaq');
-const joinUs = require('./joinUs/post');
```

Comment:
This was the endpoint for adding the data to db. 
[Question] Why did we remove it?

Reply:
This question remains unanswered. Please don't remove any existing route